generator/internal/descriptor_utils.cc

```diff
@@ -397,6 +387,27 @@ ParameterCommentSubstitution substitutions[] = {
     {R"""(`projects/<Project ID or '-'>`)""",
      R"""(``projects/<Project ID or '-'>``)"""},
 
+    // Further trim some initial paragraphs for long descriptions.
+    {R"""( The included patch
+ environment would specify the scikit-learn version as follows:)""",
+     ""},
+
```

Comment:
Is there reason to separate these substitutions, and, if so, do they deserve their own comment?

Reply:
Nope. Fixed.